Vagrantfile

```diff
@@ -21,29 +21,31 @@ Vagrant.configure("2") do |config|
 
     # install base requirements
     apt-get update
+    apt-get upgrade -y
```

Comment:
I think u are supposed to upgrade from outside (`vagrant update box`) rather then inside because upgrade might break some vagrant related setup things...